usage/jsgui/src/main/webapp/assets/js/view/catalog.js

```diff
@@ -267,7 +279,12 @@ define([
             //this.model is a constructor so it shouldn't be _.bind'ed to this
             //It actually works when a browser provided .bind is used, but the
             //fallback implementation doesn't support it.
-            var model = this.model;
+            var that = this; 
```

Comment:
yes - else if we did `this.name` in line 285 it might not resolve.  (whereas the `this.id` refers to the model's `id`.)